internal/internal_workflow_client.go

```diff
@@ -194,6 +194,11 @@ type (
 		// func which use a next token to get next page of history events
 		paginate func(nexttoken []byte) (*workflowservice.GetWorkflowExecutionHistoryResponse, error)
 	}
+
+	// queryRejectedError is a wrapper for QueryRejected
+	queryRejectedError struct {
```

Comment:
Was considering suggesting making this public, but since it's only for sending this across interceptor-to-client boundary because we can't change the interceptor signature nor the client signature, I'm ok w/ it being hidden. If a user ever wants to intercept query rejection, we can discuss making this available.